js/privatebin.js

```diff
@@ -1230,18 +1248,7 @@ jQuery.PrivateBin = function($, sjcl, Base64, RawDeflate) {
             $errorMessage = $('#errormessage');
             $loadingIndicator = $('#loadingindicator');
             $statusMessage = $('#status');
-
-            // display status returned by php code, if any (e.g. paste was properly deleted)
-            var serverStatus = $statusMessage.text();
```

Comment:
Okay, if it works this way, that's good, but the original code also included this code.